### ISSUE TITLE
Error: Cannot read property 'databaseId' of null

### ISSUE THREAD

**Issue by ajeetvolvo** [flowwer-dev/pull-request-stats#12]
Hi, my job is failing with error `Error: Cannot read property 'databaseId' of null` 

here is my workflow,

```name: Pull Request Stats

on: pull_request

jobs:
  stats:
    runs-on: ubuntu-latest
    steps:
      - name: Run pull request stats
        uses: flowwer-dev/pull-request-stats@master
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          organization: 'ajeetsingh'
          period: 30
          charts: true
          disable-links: true
          sort-by: 'REVIEWS'
```

**Comment by manuelmhtr** [comment]
Hi @ajeetvolvo could you paste the full output? I'd like to see which line is the last executed.

**Comment by manuelmhtr** [comment]
@ajeetvolvo this problem should be solved in [v2.1.0](https://github.com/flowwer-dev/pull-request-stats/releases/tag/v2.1.0). Let me know if it worked for you.

**Comment by ajeetvolvo** [comment]
yes.. its fixed now. thanks
